src/components/Calculator.js

```diff
@@ -1,170 +1,158 @@
-            onClick={this.handleClick}
-          >
-            0
-          </button>
-          <button
-            type="button"
-            className="g-item white"
-            onClick={this.handleClick}
-          >
-            .
-          </button>
-          <button
-            type="button"
-            className="g-item orange"
-            onClick={this.handleClick}
-          >
-            =
-          </button>
-        </div>
+  return (
+    <div style={{ width: '75vw', height: '75vh' }}>
+      <div className="d-grid gray">
+        <Display next={next} operation={operation} total={total} />
+        <button
+          type="button"
+          className="g-item white"
+        >
+          AC
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          +/-
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          %
+        </button>
+        <button
+          type="button"
+          className="g-item orange"
+        >
+          &divide;
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          7
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          8
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          9
+        </button>
+        <button
+          type="button"
+          className="g-item orange"
+        >
+          x
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          4
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          5
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          6
+        </button>
+        <button
+          type="button"
+          className="g-item orange"
+        >
+          -
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          1
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          2
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          3
+        </button>
+        <button
+          type="button"
+          className="g-item orange"
+        >
+          +
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+          style={{ gridColumn: '1/3' }}
+        >
+          0
+        </button>
+        <button
+          type="button"
+          className="g-item white"
+        >
+          .
+        </button>
+        <button
+          type="button"
+          className="g-item orange"
+        >
+          =
+        </button>
```

Comment:
- [ ] The first operations work as expected but then the console starts showing a loop of errors. Kindly fix them
![Screenshot from 2022-03-17 09-33-31](https://user-images.githubusercontent.com/62705947/158769833-27545629-c26e-4d9b-bcc1-9b4fde81ba60.png)